src/java.base/share/classes/java/lang/reflect/AccessFlag.java

```diff
@@ -385,12 +385,7 @@ public Set<Location> apply(ClassFileFormatVersion cffv) {
                             && PreviewFeatures.isEnabled())
                             ? Location.SET_FIELD : Location.EMPTY_SET;
                 }
-            }) {
-        @Override
-        public String toString() {
-            return "STRICT";
```

Comment:
so we want the `toString` result to be `STRICT_INIT`?

Reply:
Yes, the new spec renamed the flag to `ACC_STRICT_INIT`.